tests/export_test.py

```diff
@@ -905,13 +905,20 @@ def f_jax(x):  # x: bool[b]
       for dtype in dtypes._jax_types if dtype != np.dtype("bool")
   ])
   def test_poly_numeric_dtypes(self, dtype=np.int32):
-    if str(dtype) in {"float8_e4m3b11fnuz",
-                      "float8_e4m3fnuz",
-                      "float8_e5m2fnuz",
-                      "int2",
-                      "int4",
-                      "uint2",
-                      "uint4"}:
+    unsupported_dtypes = {
+      "float8_e4m3b11fnuz",
+      "float8_e4m3fnuz",
+      "float8_e5m2fnuz",
+      "int2",
+      "int4",
+      "uint2",
+      "uint4",
+    }
+    # TODO: Remove once minimum_jaxlib_version is 0.4.35+
+    if jax._src.lib.version < (0, 4, 35):
```

Comment:
Should this be `jax._src.lib.version >= (0, 4, 35)`?

Reply:
jaxlib 0.4.35.dev uses the latest XLA which already uses StableHLO with WEEK_4 req equal to 1.7.0. This test works fine for e3m4 and e4m3 dtypes if jaxlib-0.4.35.dev is installed.

```
>>> import jaxlib
>>> jaxlib.__version__
'0.4.35.dev20241008'
>>> import jax
>>> jax._src.lib.version
(0, 4, 35)

pytest -n auto tests/export_test.py

== 130 passed, 8 skipped in 5.67s ==

[gw37] PASSED tests/export_test.py::JaxExportTest::test_poly_numeric_dtypes_dtype_float8_e3m4
[gw37] PASSED tests/export_test.py::JaxExportTest::test_poly_numeric_dtypes_dtype_float8_e4m3

```

Reply:
btw, the following float8 types are supported in jaxlib 0.4.34 already.
```
      "float8_e4m3b11fnuz",
      "float8_e4m3fnuz",
      "float8_e5m2fnuz",
```

Will open PR to remove them from `unsupported_dtypes` in `test_poly_numeric_dtypes`